ggml/src/CMakeLists.txt

```diff
@@ -1059,7 +1108,9 @@ if (GGML_CUDA)
     list(JOIN   CUDA_CXX_FLAGS " " CUDA_CXX_FLAGS_JOINED)  # pass host compiler flags as a single argument
 
     if (NOT CUDA_CXX_FLAGS_JOINED STREQUAL "")
-        list(APPEND CUDA_FLAGS -Xcompiler ${CUDA_CXX_FLAGS_JOINED})
+        # list(APPEND CUDA_FLAGS -Xcompiler ${CUDA_CXX_FLAGS_JOINED})
+        # XXX: Removed flags: -Xcompiler
+        list(APPEND CUDA_FLAGS ${CUDA_CXX_FLAGS_JOINED})
```

Comment:
`-Xcompiler` is necessary for old versions of the CUDA toolkit.

Reply:
This was accidental. I have reverted the change.